manifests/server/instance/systemd.pp

```diff
@@ -0,0 +1,20 @@
+# @param service_name Overrides the default PostgreSQL service name.
```

Comment:
please add an `@summary` to describe what the define does.

Reply:
And consider making it `@api private`.

Reply:
does @api private work with defines?

Reply:
@summary is there

Reply:
`@api private` works also for defined resources.

Reply:
done